riscv/execute.cc

```diff
@@ -141,11 +145,71 @@ void processor_t::step(size_t n)
             state.single_step = state.STEP_STEPPED;
           }
 
+          if (debug && !state.prev_state) { // lazy init
```

Comment:
I think this code should be deleted, or at least placed in  `#if 0`.

Reply:
got it. will remove

Reply:
I remove this part and will send more complete implementation with memory trace function in future.